code/datums/quirks/negative.dm

```diff
@@ -850,3 +850,28 @@
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/very_bad_touch)
 	else
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/bad_touch)
+
+/datum/quirk/claustrophobia
+	name = "Claustrophobia"
+	desc = "You are terrified of small spaces. If you are placed inside any container, locker, or machinery, a panic attack sets in and you struggle to breath."
+	icon = "box-open"
+	value = -4
+	medical_record_text = "Patient demonstrates a fear of tight spaces."
+	hardcore_value = 5
+	processing_quirk = TRUE
+
+/datum/quirk/claustrophobia/remove()
+	SEND_SIGNAL(quirk_holder, COMSIG_CLEAR_MOOD_EVENT, "claustrophobia")
+
+/datum/quirk/claustrophobia/process(delta_time)
+	if(quirk_holder.stat != CONSCIOUS || quirk_holder.IsSleeping() || quirk_holder.IsUnconscious())
+		return
+
+	if(isturf(quirk_holder.loc))
+		SEND_SIGNAL(quirk_holder, COMSIG_CLEAR_MOOD_EVENT, "claustrophobia", /datum/mood_event/claustrophobia)
+		return 
+	
+	SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "claustrophobia")
+	quirk_holder.losebreath += 0.25 // miss a breath one in four times
+	if(prob(25))
+		to_chat(quirk_holder, span_warning("You feel trapped!  Must escape... can't breath..."))
```

Comment:
This `prob()` should be a `DT_PROB()` as it's processing on delta_time

Reply:
Thanks for pointing this out.  Fixed with https://github.com/tgstation/tgstation/pull/63741/commits/079f3b3eaa042830634436788436ffe0cf737d1f